master/security/service-accounts.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Use service accounts in policy rules
+---
+
+### Big picture
+
+Use Calico network policy to allow/deny traffic for Kubernetes services. 
+
+### Value
+
+Using Calico network policy, you can leverage Kubernetes service accounts with RBAC for flexible control over how policies are applied in a cluster. For example, the network security can have RBAC permissions to:
+
+- Control which service accounts the developer team can use within a namespace
+- Write high-priority network policies for those service accounts (that the developer team cannot override) 
+
+The network security team can maintain full control of security, while selectively allowing developer operations where it makes sense.  
```

Comment:
Same here - I'm missing how this is connected to service accounts at all (or how this would be effectively done using Calico).